texastribunestyle.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: stylepage
+title: TexasTribune
+typekit: brb4jmm
+css: texastribune.min.css
+---
+
+
+<!-- masthead -->
+<div class="masthead sixteen columns">
+  <img class="logo" src="https://s3.amazonaws.com/static.texastribune.org/common/images/logo.jpg" title="TexasTribune">
+
+Stylesheet
+==========
+{: .sg-header}
+
+This is the styleguide for [texastribune.org](http://www.texastribune.org).
+Below are styles that we're currently using that we'll continue to iterate on
+and use in the redesign of the site planned for 2015.
+
+</div><!-- end masthead -->
+
+<div class="four columns"><!-- side nav -->
+  <div id='side-nav'>
+
+* <a href="#sass">Sass/CSS</a>
+* <a href="#javascript">JavaScript</a>
+* <a href="#branding">Branding</a>
+* <a href="#typography">Typography</a>
+* <a href="#color">Color</a>
+* <a href="#grid">Grid</a>
+* <a href="#navigation">Navigation</a>
+* <a href="#images">Images</a>
+* <a href="#bylines--credits">Bylines & Credits</a>
+* <a href="#paragraphs">Paragraphs</a>
+* <a href="#lists">Lists</a>
+* <a href="#forms">Forms</a>
```

Comment:
iduno :/ I left it incomplete because I couldn't decide which I preferred. I did some research to see if there was an even terser syntax but could not find one. I vote for converting them all to markdown style.